library.properties

```diff
@@ -1,10 +1,10 @@
-name=Arduino_LSM6DSOX
+name=Arduino_LSM6DSOX (torenrl fork)
 version=1.0.0
-author=Arduino
+author=Arduino -> Tore
 maintainer=Arduino <info@arduino.cc>
 sentence=Allows you to read the accelerometer and gyroscope values from the LSM6DSOX IMU on your Arduino Nano RP2040 Connect.
 paragraph=
 category=Sensors
-url=https://github.com/arduino-libraries/Arduino_LSM6DSOX
+url=https://github.com/torenrl/Arduino_LSM6DSOX
```

Comment:
Hi @torenrl. It's not clear what your intention is here.

You have submitted a pull request to Arduino's repository, which is done in order to propose that a change be made here in the official library. But then you have made changes to the [library metadata](https://arduino.github.io/arduino-cli/latest/library-specification/#library-metadata) here that could never be accepted into the official library (and even make the library invalid in the case of the parentheses introduced by the name change), which make it seem that your intention is to maintain your own fork of the library.

Reply:
Hi @per1234 

Thank you for the comments. It is not my intention to maintain my own fork, and the pull request was because I think others might benefit from the changes I made to the library. However, does this mean that I should remove the changes to the metadata, and make a new pull request, or is it possible to change this request somehow?

Reply:
Any commit you push to the `master` branch of your fork is added to this pull request. So you are free to make changes.

Reply:
Ok, great. I now reversed the metadata in my fork. I hope the changes otherwise looks good, I have tried to maintain the coding style originally used in the sources.